### ISSUE TITLE
Fix travis

### ISSUE THREAD

**Issue by mariobehling** [fossasia/apps.loklak.org#53]
Travis has not worked in thei repo yet. Please help to make it run.

**Comment by adarsh-ideal** [comment]
@mariobehling I would like to work on this.

**Comment by adarsh-ideal** [comment]
@hemantjadon Which tests should I add in the script?

**Comment by Abhi2424shek** [comment]
@adarsh my suggestions: Use coala with:
 - HTML Lint Bear
- CSS Lint Bear
- JavaScript Lint Bear
- Line Length Bear

**Comment by adarsh-ideal** [comment]
@Abhi2424shek Thanks Abhishek! I am not adding Line Length Bear currently.

**Comment by mariobehling** [comment]
Not yet working.

**Comment by adarsh-ideal** [comment]
JS lint bear is causing to exceed memory limit. I'll remove it.

**Comment by adarsh-ideal** [comment]
Linters are working now.

**Comment by jig08** [comment]
TravisCI Build needs to be fixed.

**Comment by adarsh-ideal** [comment]
@jig08 Can you please describe a little what to do?

**Comment by Abhi2424shek** [comment]
@adarsh-ideal it would be best to check the log of Travis CI.

**Comment by adarsh-ideal** [comment]
@jig08 There are linting errors in the build log, caused by linters I added for testing in .coafile. This means that travis is working perfectly.

This issue was to
https://github.com/fossasia/apps.loklak.org/issues/53#issue-199690808

And to solve linting errors we could create a new issue mentioning "Fix linting errors."

**Comment by rhemon** [comment]
its good travis is fixed, but it brought way too much errors, and I think it needs to be fixed. either we ignore errors (which is probably not the best thing to do) or we fix them (which is probably the hardest thing to do).

**Comment by rhemon** [comment]
created an issue for it https://github.com/fossasia/apps.loklak.org/issues/82

**Comment by adarsh-ideal** [comment]
@rhemon Thanks, I'll solve it as soon as my last task completes.

**Comment by rhemon** [comment]
ok no probs. but before starting we should divide the work since I think dealing with it alone will be very difficult (unless u want to :p ) .  @adarsh-ideal

**Comment by daminisatya** [comment]
@rhemon will divide the work and fix this. Build failures needs to be tackled immediately.

**Comment by adarsh-ideal** [comment]
@rhemon There are two linters HTML & CSS.
I am solving html errors.

**Comment by rhemon** [comment]
alright. go on. i will start tonight

**Comment by adarsh-ideal** [comment]
@rhemon You could solve CSS errors

**Comment by rhemon** [comment]
@adarsh-ideal how far are you with html errors? 
the css part was quite small, although thats probably because it has lots of bear missing
https://github.com/rhemon/apps.loklak.org/pull/3
so I am thinking of adding those bears, also bears for JavaScript, but I think these are later part, presently we should fix up the present things with the bears that are added and get it merged in this repo so that other. so once you are done let me know, we will merge our changes together and then do one PR here.

**Comment by adarsh-ideal** [comment]
@rhemon It's almost done, I have an assignment to complete. Will make the PR tomorrow.

**Comment by rhemon** [comment]
@adarsh-ideal  before doing PR to this branch it will be better if we merge our forks, and then do one PR together making sure that its fully green, if you want you can just do this by fetching my branch(`travis-fix-css`) and then cherry-pick the commit into your one. then do a PR

**Comment by Abhi2424shek** [comment]
@rhemon It would be better if you could PR to his fork, and he merges It and then he makes a PR to here, or he makes a PR to your fork, you merge it and you make the PR to this repository, provided the forks of both of you are ready with your work @adarsh-ideal

**Comment by rhemon** [comment]
both are same tbh. doesnt really matter how you do it. it just reduces one of our work :p. but no probs, i will do the PR to adarsh's fork (@adarsh-ideal let me know the branch you are working on) when I get access to the computer. 
this friday is my sister's wedding so things are quite busy at home :)

**Comment by daminisatya** [comment]
@rhemon You can create a PR fixing from your fork! :) and if things doesn't work i will assign the issue back to someone else! thank you so much for the work so far.

**Comment by rhemon** [comment]
ok give me some time. presently as i mentioned earlier things are busy at
home so its hard to sit on computer.

**Comment by adarsh-ideal** [comment]
@rhemon @daminisatya Fixed all html lint errors.
Got a a green build ( https://travis-ci.org/adarsh-ideal/apps.loklak.org/builds/193485360 )
when disabled CSS lint.

It has some merge conflicts with base branch, I'll resolve them and will make a pull request.

**Comment by adarsh-ideal** [comment]
@rhemon This is my repo https://github.com/adarsh-ideal/apps.loklak.org/
Branch: html-fix

**Comment by adarsh-ideal** [comment]
@mariobehling @daminisatya I think we should close this issue, travis is working now.
We should follow-up on https://github.com/fossasia/apps.loklak.org/issues/82 to reslove errors.
